playgame.py

```diff
@@ -24,6 +25,24 @@ def main(argv):
     else:
         run_one_game(args)
 
+def create_dirs(default,path):
```

Comment:
This can be pretty nicely reduced. os has a makedirs function which recursively makes directories in a path. You will have to check the path exists.
If yes, continue as before.
If not, create the path using makedirs.
Looks like 2 lines, and good to still have in a function (though without a default argument)